argo-cd-apps/app-of-app-sets/environment/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../base
+namespace: konflux-public-staging
+patches:
+  - target:
+      kind: Application
+      name: all-application-sets
+    patch: |-
+      - op: replace
+        path: /spec/source/path
+        value: argo-cd-apps/overlays/environment
```

Comment:
Using `environment` for the overlay name is likely to generate some confusion. It's an overloaded term when our other overlays are named after deployment environments (e.g. `development`, `production-downstream`). Maybe we should use `eaas` unless someone else has a better suggestion.

Also, should this name include a `-downstream` postfix in case we create an equivalent public overlay later?

Reply:
This change can be closed now, here is the replacement: https://github.com/redhat-appstudio/infra-deployments/pull/3675